### DISCUSSION TITLE
FIrefoxESR 115.10.0 not displaying videodownloadhelper icon

### DISCUSSION THREAD

**Discussion by zeekisthename** [aclap-dev/video-downloadhelper#428] [Help and Q&A]
### Your question

Noted version of FIrefox for Linux does not display (installed) videodownloadhelper icon in browser toolbar in order to click it to enable downloads.

Attempted to find icon under the Customize Toolbar in order to drag it to toolbar, but it did not appear to be there.

Is there a problem with FIrefox ESR showing the icon, as it appears in regular Firefox for Linux 125.0.2?

Where, if anywhere is the icon displayed?

### Addon & CoApp version and details

videodownloadhelper version 8.2.2.8 stable
companion app: VdhCoApp 2.0.19

There are no matching discussions.

You can open a[ new discussion ](https://github.com/aclap-dev/video-downloadhelper/discussions/new)to ask questions about this repository or get help.

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user, like you.

Try clicking the jigsaw puzzle piece in the upper right area of the browser window.

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/5572ac28-2aec-4c35-92e1-626185352c3c)

VDH might be listed there.  If it is, click its gear icon, then pin it to the toolbar.  Might work, might not, has worked for a lot of people.

But why are you stuck on that old ESR version of Firefox?  The latest is 125.0.2.  Are you running Windows 7?

**Reply by GrampaWildWilly** [reply]
Oops.  Stupid question.  You said you're on Linux.  In that case, I don't know why you would intentionally keep yourself on such an outdated version.  But then, I don't know squat about Linux.

**Comment by zeekisthename** [comment]
Thank you for your reply.

Running Linux with 125.0.2 with one config, and ESR with another config for backup.

Sent with [Proton Mail](https://proton.me/) secure email.
